test/flipper_app/authorize_test.rb

```diff
@@ -0,0 +1,75 @@
+require_relative '../test_helper'
+require 'mocha/setup'
+require 'flipper_app/authorize'
+
+class AssignmentsApiTest < Minitest::Test
+  def test_unauthenticated_users_rejected
+    User.stubs(:where).returns(authorized_users)
```

Comment:
What do you think about this next version, where I'm stubbing the `User ActiveRecord::Relation`?